manageiq-gems-pending.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |s|
   s.executables   = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
   s.require_paths = ["lib", "lib/gems/pending", "lib/gems/pending/util"]
 
-  s.required_ruby_version = "> 2.2.2"
+  s.required_ruby_version = "> 2.3.1"
 
-  s.add_runtime_dependency "actionpack",              ">= 5.0", "< 5.2"
-  s.add_runtime_dependency "activerecord",            ">= 5.0", "< 5.2"
-  s.add_runtime_dependency "activesupport",           ">= 5.0", "< 5.2"
+  s.add_runtime_dependency "actionpack",              "< 5.3"
+  s.add_runtime_dependency "activerecord",            "< 5.3"
+  s.add_runtime_dependency "activesupport",           "< 5.3"
```

Comment:
LGTM here

Reply:
curiosity: do we even need actionpack here?

```
05:11:49 ~/Code/manageiq-gems-pending (master) (2.4.6) + git grep -E "Action|Abstract"
lib/gems/pending/util/miq-exception.rb:  # Abstract MIQ Error Class
lib/gems/pending/util/miq-exception.rb:  # Concrete MIQ Policy Action Errors/Exceptions based on abstract Error class above
lib/gems/pending/util/miq-exception.rb:  class MiqActionError < Error; end
lib/gems/pending/util/miq-exception.rb:  class StopAction < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class AbortAction < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class UnknownActionRc < MiqActionError; end
lib/gems/pending/util/miq-exception.rb:  class PolicyPreventAction < MiqActionError; end
lib/gems/pending/util/win32/miq-powershell.rb:    #     $_ | Remove-Variable -Scope Local -ErrorAction SilentlyContinue
lib/gems/pending/util/win32/miq-psd.ps1:$ErrorActionPreference = "SilentlyContinue"
```

I think we pulled in it in 17196b6cbfa6de6681eb7769b68b251e9f5a2df3 , then 2b278f477a959c943ef6269e1eb93ea72596aa85, and finally f6704e20161a08f8a0683f98262290fb25a13e50

Reply:
I don't know, shall I try removing it and see if it still works?